GiniVision/Assets/Localizable.strings

```diff
@@ -27,6 +27,8 @@
 "ginivision.camera.documentValidationError.tooManyPages" = "Das Dokument darf nicht mehr als 10 Seiten haben";
 "ginivision.camera.documentValidationError.wrongFormat" = "Nur Dateien vom Format jpg, tiff, png, gif oder pdf können analysiert werden";
 "ginivision.camera.fileImportTip" = "Es können jetzt auch ganz einfach Dateien hochgeladen werden.";
+"ginivision.camera.qrCodeDetectedPopup.message" = "QR detected!";
```

Comment:
Shouldn't that be in german?

Reply:
Yes, I was waiting for the german version that is why I left these in english.